pages/security.md

```diff
@@ -0,0 +1,23 @@
+---
+layout: default
+title: Security
+main_nav_id: security
+---
+
+# Kotlin Security
+
+We do our best to make sure our products are free of security vulnerabilities. To reduce the risk of introducing a vulnerability,
+you can follow these best practices: 
+
+* Always use the latest Kotlin release. For security purposes, we sign our releases published on [Maven Central](https://search.maven.org/search?q=g:org.jetbrains.kotlin) 
+with these PGP keys.
+* Use the latest versions of your application’s dependencies. If you still need to use a specific dependency version, 
```

Comment:
```suggestion
* Use the latest versions of your application’s dependencies. If you need to use a specific version of a dependency, 
```

Reply:
`still` looks redundant: if developers have their reasons to use the older versions, we can't affect it.